### DISCUSSION TITLE
Budget Category Width

### DISCUSSION THREAD

**Discussion by DPSever** [actualbudget/releases#30] [Feature ideas]
The category column in the budget screen is currently a fixed width (200px) no matter the available screen real-estate. Can this be made adjustable to show more of the category title? At a minimum for a quicker fix could the "title" attribute be set so you could see the full category name on hover? At the moment to see longer names you have to click the dropdown arrow and then select "rename" to see the full name.

**Comment by bpaulien** [comment]
I'm coming across this issue on several of my category names.  Besides the solutions mentioned in the OP, also it might be possible to just "word wrap" the field, so you could see it all.  Granted, that might make the budget look strange, with taller fields for some rows. Personally, I think at a minimum seeing the whole thing by hovering would be ok.  I have a couple that are much longer than the rest, and I wouldn't necessarily want the entire column to be huge just for the sake of a minority of longer category names, but it's really inconvenient to not be able to see the name without going into the "rename" option.

**Comment by kazzazed** [comment]
Adding to this one, if the budget name is too long, you can't get to the settings cog on the right of the budget name. If the column was expandable that would solve that issue.
